integration/setup/config.go

```diff
@@ -87,8 +87,7 @@ func NewConfig() (Config, error) {
 		c := framework.HostConfig{
 			Logger: logger,
 
-			ClusterID:  env.ClusterID(),
-			VaultToken: env.VaultToken(),
+			ClusterID: env.ClusterID(),
```

Comment:
The Vault token got dropped because the legacy framework does not do anything with it.